webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
```

Comment:
I checked - it seems that the only objects that have a reference to the MediaStreamTrack at the moment are the stream object and the RTPStream object. If we're OK with sending only those two objects, I think listing them explicitly is no great burden.